### ISSUE TITLE
hardcoded path `/usr/libexec/xenopsd/igmp_query_injector.py`

### ISSUE THREAD

**Issue by ydirson** [xapi-project/xen-api#5446]
`gmp_query_injector.py` install location can be parametrized, but the caml code will only ever look at it in a single place:

```
Makefile:       install -D ./ocaml/xenopsd/scripts/igmp_query_injector.py $(DESTDIR)/$(XENOPSD_LIBEXECDIR)/igmp_query_injector.py
ocaml/networkd/lib/network_utils.ml:let inject_igmp_query_script = ref "/usr/libexec/xenopsd/igmp_query_injector.py"
ocaml/xenopsd/xc/xc_resources.ml:  ref "/usr/libexec/xenopsd/igmp_query_injector.py"
```

**Comment by psafont** [comment]
The default value has never been used by xenserver, it's been broken since its introduction: https://github.com/xapi-project/xenopsd/commit/056854d0a0cbd269fa5aadf6ee5d07558f29afbe#diff-90d08e583c4c9c6f391b2ae90f819f600a6326928ea9512c9e0c6d98e9f29ac2R21

Easiest way to deal with it is to change the value in configure.ml to match the one used in xenserver

**Comment by ydirson** [comment]
> Easiest way to deal with it is to change the value in configure.ml to match the one used in xenserver

Yes, but that would only fix part of the problem - should it be made non-configurable as well?
